fitbenchmarking/utils/misc.py

```diff
@@ -31,21 +31,28 @@
 from utils import user_input
 
 
-def get_minimizers(software):
+def get_minimizers(software, minimizers_list, json_file):
```

Comment:
as far as I can tell you always use this function after extracting the arguments from `software_options`. It would be neater to just pass the `software_options` and extract it here